asp-core/src/main/java/de/jcup/asp/core/CryptoAccess.java

```diff
@@ -72,7 +75,7 @@ public String encrypt(String strToEncrypt) {
         try {
             Cipher cipher = Cipher.getInstance(SECURITY_ALGORITHM);
             cipher.init(Cipher.ENCRYPT_MODE, secretKey, new SecureRandom());
-            byte[] encryptedBytes = cipher.doFinal(strToEncrypt.getBytes("UTF-8"));
```

Comment:
I think we could remove dependency to `apache commons` here but still solve the problem, if we would do following:

My fault was to create the cypher bytes by UTF-8 but on decryption time id did only use the default encoding of the JVM (what is at my machine UTF-8 but not everywhere...).

I would suggest to keep:
```java
 byte[] encryptedBytes = cipher.doFinal(strToEncrypt.getBytes("UTF-8")); // as done before PR
```
but instead of 

```java
return new String(decryptedBytes);
```
we could use 

```java
return new String(decryptedBytes,"UTF-8");
```

For the cipher these are just bytes - no matter which kind of encoding is used. But we must ensure input and output string encoding is same.
I think this would solve the problem and we could reduce dependency to another library (commons-text).

What do you think - do you agree ?

Reply:
Looks like you are right `new String(decryptedBytes,"UTF-8");` is enough.